cmake/ExternalProjects.cmake

```diff
@@ -31,12 +31,18 @@ set(GRPC_PLUGIN /usr/local/bin/grpc_cpp_plugin)
 
 # Pistache 
 ExternalProject_Add(pistache_ext
-    GIT_REPOSITORY "https://github.com/oktal/pistache.git"
+    GIT_REPOSITORY "https://github.com/pistacheio/pistache.git"
     GIT_TAG "2ef937c434810858e05d446e97acbdd6cc1a5a36"
     CMAKE_CACHE_ARGS "-DCMAKE_INSTALL_PREFIX:STRING=${CMAKE_INSTALL_PREFIX}"
+    BUILD_BYPRODUCTS ${CMAKE_INSTALL_PREFIX}/lib/libpistache.so
 )
 ExternalProject_Get_Property(pistache_ext SOURCE_DIR)
 set(PISTACHE_INCLUDE_DIR ${SOURCE_DIR}/include)
+add_library(pistache SHARED IMPORTED)
+add_dependencies(pistache pistache_ext)
+set_target_properties(pistache
+    PROPERTIES IMPORTED_LOCATION ${CMAKE_INSTALL_PREFIX}/lib/libpistache.so
+)
```

Comment:
Nice, this looks relatively clean. Can you move faasm and faasm-toolchain to this version of Faabric to check they both still work after (or before) merging this?

Reply:
Here is [faasm's](https://github.com/faasm/faasm/pull/388/checks?check_run_id=1733786695) and here the [toolchain's](). Should we merge those PRs?

Reply:
Here is [faasm's](https://github.com/faasm/faasm/pull/388/checks?check_run_id=1733786695) and here the [toolchain's](https://github.com/faasm/faasm-toolchain/pull/25/checks?check_run_id=1734076349). Should we merge those PRs?

Note that the toolchain one would not pass for some missmatch in the submodule build (which I am looking into) but afaik everything else worked fine (as to what this PR is concerned).

Reply:
@Shillaker ok for merge?